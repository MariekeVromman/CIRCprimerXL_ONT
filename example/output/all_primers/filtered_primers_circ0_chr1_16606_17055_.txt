circ0	chr1	16606	17055	0	TGGCACATAGAAGTAGTTCTC	CCACTCCCACCCTCAA	123	21	228	16	58.233	58.477	42.857	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	4	TGGCACATAGAAGTAGTTCTCT	CCACTCCCACCCTCAA	123	22	228	16	59.723	58.477	40.909	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	1	CTGGCACATAGAAGTAGTTCT	CCACTCCCACCCTCAA	122	21	228	16	58.229	58.477	42.857	62.500	CTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	3	CATGAGGTCGTTGGCAAT	AGCGCGTCTCCTTCAG	48	18	165	16	59.074	59.965	50.000	62.500	CATGAGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCT	PASS
circ0	chr1	16606	17055	5	GCAGGTCAGGCAGGTA	CCACTCCCACCCTCAA	71	16	228	16	59.052	58.477	62.500	62.500	GCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	6	AGGTCGTTGGCAATGC	CCACTCCCACCCTCAA	52	16	228	16	58.725	58.477	56.250	62.500	AGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	2	ATGAGGTCGTTGGCAATG	CCACTCCCACCCTCAA	49	18	228	16	59.074	58.477	50.000	62.500	ATGAGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
