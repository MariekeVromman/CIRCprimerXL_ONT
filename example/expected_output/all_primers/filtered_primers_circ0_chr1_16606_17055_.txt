circ0	chr1	16606	17055	0	TGGCACATAGAAGTAGTTCTC	CCACTCCCACCCTCAA	124	21	229	16	58.233	58.477	42.857	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	1	CTGGCACATAGAAGTAGTTCT	CCACTCCCACCCTCAA	123	21	229	16	58.229	58.477	42.857	62.500	CTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	2	ATGAGGTCGTTGGCAATG	CCACTCCCACCCTCAA	50	18	229	16	59.074	58.477	50.000	62.500	ATGAGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	3	CATGAGGTCGTTGGCAAT	CCACTCCCACCCTCAA	49	18	229	16	59.074	58.477	50.000	62.500	CATGAGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	4	TGGCACATAGAAGTAGTTCTCT	CCACTCCCACCCTCAA	124	22	229	16	59.723	58.477	40.909	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	5	GCAGGTCAGGCAGGTA	CCACTCCCACCCTCAA	72	16	229	16	59.052	58.477	62.500	62.500	GCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ0	chr1	16606	17055	6	AGGTCGTTGGCAATGC	CCACTCCCACCCTCAA	53	16	229	16	58.725	58.477	56.250	62.500	AGGTCGTTGGCAATGCCGGGCAGGTCAGGCAGGTAGGATGGAACATCAATCTCAGGCACCTGGCCCAGGTCTGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
